SU-SWS/stanford_basic
# Latest Decanter

# READY FOR REVIEW

# Summary
- Address NPM vulnerabilities
- Pull in latest Decanter

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1.Pull this branch
2. Verify that the latest Decanter has been pulled into `libraries/decanter`

# Affected Projects or Products
- stanford_basic

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
